src/sas/qtgui/Perspectives/Fitting/ComplexConstraint.py

```diff
@@ -291,14 +331,31 @@ def onApply(self):
         if self.parent.constraint_accepted:
             self.setupParamWidgets()
 
+    def applyAcrossTabs(self, tabs, param, expr):
+        """
+        Apply constraints across tabs, e.g. all `scale` parameters
+        constrained to an expression. *tabs* is a list of active fit tabs
+        for which the parameter string *param* will be constrained to the
+        *expr* string.
+        """
+        for tab in tabs:
+            if hasattr(tab, "kernel_module"):
+                if param in tab.kernel_module.params:
+                    constraint = Constraint(param=param, value=param,
+                                            func=expr,
+                                            value_ex=tab.kernel_module.name +
+                                            "." + param, operator="=")
```

Comment:
Bad line break. You are breaking in the middle of an expression, and adding a new parameter after the expression on the next line. You could resolve this by breaking after open paren so that you have more room on the line,  or by preassigning value_ex to an expression so that it isn't mixed up with the arguments. If you do need to break an expression, break before the binary operator rather than after.